plastexdepgraph/Packages/depgraph.py

```diff
@@ -29,6 +29,10 @@
   input a plasTeX node and outputting CSS colors for the boundary and interior
   of graph nodes respectively.
 
+* document.userdata['dep_graph']['stylerizer'] can be a function taking as input
+  a plasTeX node and outputting a graphviz style
+  (see https://graphviz.org/docs/attr-types/style/)
```

Comment:
It is a bit weird to group the first two and leave the third one alone. I guess grouping all would be hard to read, so could you ungroup the first two (and add a final period at the end of your sentence)?

Reply:
Sure thing! How does this look?